coordinator/src/routes/dispense/release.ts

```diff
@@ -44,22 +26,19 @@ export default [
           .code(400)
       }
 
-      request.logger.info("Sandbox release response")
-      const messageId = "d5a20db9-6d76-4aeb-a190-9a85843b01bf"
-      const sandboxResponse: fhir.Bundle = {
-        resourceType: "Bundle",
-        type: "searchset",
-        total: 2,
-        id: messageId,
-        identifier: {
-          value: messageId
-        },
-        entry: [
-          {resource: genericBundle1, fullUrl: `urn:uuid:${bundle1Id}`},
-          {resource: genericBundle2, fullUrl: `urn:uuid:${bundle2Id}`}
-        ]
-      }
-      return responseToolkit.response(sandboxResponse)
+      const payloadAsParameters = requestPayload as fhir.Parameters
+
+      request.logger.info("Building Spine release request")
+      const spineRequest = translator.convertParametersToSpineRequest(payloadAsParameters)
+      spineRequest.messageId = request.headers["nhsd-request-id"].toUpperCase()
+
+      request.logger.info("Awaiting response")
```

Comment:
This kind of logging might be better inside the spine communication handler

Reply:
Copied this from the process message route. I'll move in both places